installation/README.md

```diff
@@ -3,10 +3,11 @@
 ## Logging - Bash Script output rules
 
 ```bash
-print_lc "This message will be logged to both console and logfile"
-print_c  "This message will only be logged to the console"
-log      "This message will only be logged to the logfile"
-clear_c  "Clears the console screen"
+print_lc    "This message will be logged to both console and logfile"
+print_c     "This message will only be logged to the console"
+log         "This message will only be logged to the logfile"
+run_and_log "Run a command and log its output to logfile"
```

Comment:
Wrong description. its actually doing "console and logfile"

Reply:
Changed with the next iteration.

Reply:
Next one coming or the 'rename function to run_and_print_lc' commit, as is wasnt changed there.